infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/FsmTaskManager.java

```diff
@@ -26,9 +26,13 @@
 
 /**
  * This class manages the tasks submitted by the runtime FSM, the replication FSM and for the sink tasks when SINK is
```

Comment:
It will also be useful to add that this class is a singleton and only 1 instance per LR process is created.